go.sum

```diff
@@ -273,8 +273,8 @@ github.com/newrelic/go-agent/v3 v3.20.3 h1:hUBAMq/Y2Y9as5/yxQbf0zNde/X7w58cWZkm2
 github.com/newrelic/go-agent/v3 v3.20.3/go.mod h1:rT6ZUxJc5rQbWLyCtjqQCOcfb01lKRFbc1yMQkcboWM=
 github.com/newrelic/go-insights v1.0.3 h1:zSNp1CEZnXktzSIEsbHJk8v6ZihdPFP2WsO/fzau3OQ=
 github.com/newrelic/go-insights v1.0.3/go.mod h1:A20BoT8TNkqPGX2nS/Z2fYmKl3Cqa3iKZd4whzedCY4=
-github.com/newrelic/newrelic-client-go/v2 v2.21.1 h1:F00Sv41LTSm8edt1Ld2qnV7Jw0lRee8t921odyaK3nc=
-github.com/newrelic/newrelic-client-go/v2 v2.21.1/go.mod h1:VPWTvEfKvnTZLunAC7fiW33y4e0srznNfN5HJH2cOp8=
+github.com/newrelic/newrelic-client-go/v2 v2.21.2-0.20231011094303-bbae780fcced h1:Qk2TPqUXjQEoZGFNvgHFD03riHzYUTb+jZjQq8gsPbI=
+github.com/newrelic/newrelic-client-go/v2 v2.21.2-0.20231011094303-bbae780fcced/go.mod h1:VPWTvEfKvnTZLunAC7fiW33y4e0srznNfN5HJH2cOp8=
```

Comment:
Please revert this change before you merge the PR, @pranav-new-relic 
This change was added to allow compilation/integration tests to succeed as they would need to point to the changes in the Go-Client counterpart of this PR.